python/kvikio/nvcomp_codec.py

```diff
@@ -66,18 +66,18 @@ def encode(self, buf):
         """
         return self.encode_batch([buf])[0]
 
-    def encode_batch(self, bufs: List[Any]) -> List[Any]:
+    def encode_batch(self, bufs: Sequence[Any]) -> Sequence[Any]:
         """Encode data in `bufs` using nvCOMP.
 
         Parameters
         ----------
-        bufs : List[buffer-like].
+        bufs : Sequence[buffer-like].
             Data to be encoded. Each buffer in the list may be any object
             supporting the new-style buffer protocol.
 
         Returns
         -------
-        enc : List[buffer-like]
+        enc : Sequence[buffer-like]
             List of encoded buffers. Each buffer may be any object supporting
             the new-style buffer protocol.
```

Comment:
```suggestion
            List of encoded buffers. Each buffer may be any object supporting
            the new-style buffer protocol.
```

Reply:
Just a quick note that I've tried to follow the style from `numcodecs.abc.Codec` even though I think it's too verbose (i.e. I agree with your comment).

Reply:
Done.